cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc

```diff
@@ -335,6 +335,16 @@ Replaced by:
 TERMINATE TRANSACTION[S] transaction-id[,...]
 ----
 
+a|
+label:functionality[]
+label:removed[]
+[source, cypher, role="noheader"]
+----
+SHOW TRANSACTIONS YIELD allocatedBytes
+----
+a|
+This column has been removed as it wasn't tracked and always returned 0.
+
```

Comment:
Did you mean to include this? Doesn't seem related to Octal or Hex

Reply:
Actually I just moved it up two steps to be together with the other SHOW TRANSACTION entries, but looks like Github is not presenting it very well

Reply:
Oh yes my bad :)